graphics/test/image_unittest.cc

```diff
@@ -123,6 +124,10 @@ TEST(ImageTest, Drawing) {
       EXPECT_EQ(image.GetBlue(i, j), 0);
     }
   }
+  std::vector<int> points = {20, 20, 20, 22, 22, 21};
+  image.DrawPolygon(points, red);
```

Comment:
can you add some polygon-speicifc tests: first, that drawing a polygon clockwise is the same as drawing the same thing counterclockwise? second, if drawing a rect is the same as drawing a polygon with four points?